packages/core/admin/admin/src/content-manager/utils/schema.js

```diff
@@ -217,29 +217,16 @@ const createYupSchemaAttribute = (type, validations, options) => {
   }
 
   if (type === 'blocks') {
-    schema = yup
-      .mixed(errorsTrads.json)
-      .test('isJSONObject', errorsTrads.json, (value) => {
-        try {
-          const parsedValue = JSON.parse(value);
-
-          if (typeof parsedValue === 'object' && parsedValue !== null) {
-            return true;
-          }
-        } catch (err) {
-          return false;
-        }
-
-        return false;
-      })
-      .nullable()
-      .test('required', errorsTrads.required, (value) => {
-        if (validations.required && (!value || !value.length)) {
-          return false;
-        }
+    // TODO: Fix frontend validation for blocks, what should we check here?
```

Comment:
I'm not sure it's worth doing more than that on the frontend given that the json can only be obtained from interacting with the editor